src/beta_inc.jl

```diff
@@ -930,7 +930,8 @@ function _beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64=1-p)
     x = p
     r = sqrt(-2*log(p))
     p_approx = r - @horner(r, 2.30753e+00, 0.27061e+00) / @horner(r, 1.0, .99229e+00, .04481e+00)
-    fpu = 1e-30
+    sae = log10(floatmin(Float64))
+    fpu = 10^sae
```

Comment:
```suggestion
    fpu = exp10(sae)
```
with this change, it looks like LLVM can constant-fold the calculation of `fpu`.